external/windows64-gcc/README.md

```diff
@@ -91,37 +75,11 @@ rem Build and install NuPIC.core, and build SWIG binding libraries
 mingw32-make -f Makefile install
 
 rem Install Python SWIG nupic.bindings
-cd ..\..
-python setup.py install
-</pre>
```

Comment:
Instead of putting all this in a `<pre/>` block, you should use the standard markdown for [fenced code blocks](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks) (here and elsewhere).